src/main/resources/topics-and-qrels/topics.ntcir8zh.eval.txt

```diff
@@ -0,0 +1,73 @@
+ACLIA2-CS-0002	《千里走单骑》和张艺谋是什么关系？
```

Comment:
Can we make the qrels and topics have the same name? Why `ntcir8zh` vs. `nctir8` above?